src/test/java/teammates/test/cases/BaseComponentTestCase.java

```diff
@@ -106,9 +106,21 @@ protected static void verifyPresentInDatastore(FeedbackQuestionAttributes expect
         if (wildcardId) {
             expected.setId(actual.getId());
         }
+        
+        setEqualTimestampBetweenExpectedAndActual(expected, actual);
+        
```

Comment:
Rather than setting the expected timestamp to be equals to theactual timestamp, what we can do is to add the keyword "transient" to the createdAt and updatedAt attributes in the entity class. This allows gson to ignore it when converting to json. There will be no need to have this function then.
Sorry for not commenting about this sooner. I only realised that I could use the transient keyword after my pr, and forgot to mention it.